vi/cheatsheet-supervised-learning.md

```diff
@@ -0,0 +1,567 @@
+
+&#10230; Các phương thức này có thể được sử dụng cho cả bài toán hồi quy lẫn bài toán phân loại.
+
+<br>
+
+**65. CART ― Classification and Regression Trees (CART), commonly known as decision trees, can be represented as binary trees. They have the advantage to be very interpretable.**
+
+&#10230; CART - Cây phân loại và hồi quy (CART), thường được biết đến là cây quyết định, có thể được biểu diễn dưới dạng cây nhị phân. Chúng có các ưu điểm có thể được diễn giải một cách dễ dàng.
+
+<br>
+
+**66. Random forest ― It is a tree-based technique that uses a high number of decision trees built out of randomly selected sets of features. Contrary to the simple decision tree, it is highly uninterpretable but its generally good performance makes it a popular algorithm.**
+
+&#10230; Rừng ngẫu nhiên - Là một kĩ thuật dựa trên cây (tree-based), sử dụng số lượng lớn các cây quyết định để lựa chọn ngẫu nhiên các tập thuộc tính. Ngược lại với một cây quyết định đơn, kĩ thuật này khá khó diễn giải nhưng do có hiệu năng tốt nên đã trở thành một giải thuật khá phổ biến hiện nay.
+
+<br>
+
+**67. Remark: random forests are a type of ensemble methods.**
+
+&#10230; Chú ý: rừng ngẫu nhiên là một loại giải thuật ensemble.
+
+<br>
+
+**68. Boosting ― The idea of boosting methods is to combine several weak learners to form a stronger one. The main ones are summed up in the table below:**
+
+&#10230; Boosting - Ý tưởng của các phương thức boosting là kết hợp các phương pháp học yếu hơn để tạo nên phương pháp học mạnh hơn. Những phương thức chính được tổng kết ở bảng dưới đây:
+
+<br>
+
+**69. [Adaptive boosting, Gradient boosting]**
+
+&#10230; [Adaptive boosting, Gradient boosting]
+
+<br>
+
+**70. High weights are put on errors to improve at the next boosting step**
+
+&#10230; Các trọng số có giá trị lớn được đặt vào các phần lỗi để cải thiện ở bước boosting tiếp theo
+
+<br>
+
+**71. Weak learners trained on remaining errors**
+
+&#10230; Các phương pháp học yếu huấn luyện trên các phần lỗi còn lại 
+
+<br>
+
+**72. Other non-parametric approaches**
+
+&#10230; Các cách tiếp cận phi-tham số khác
+
+<br>
+
+**73. k-nearest neighbors ― The k-nearest neighbors algorithm, commonly known as k-NN, is a non-parametric approach where the response of a data point is determined by the nature of its k neighbors from the training set. It can be used in both classification and regression settings.**
+
+&#10230; k-nearest neighbors - Giải thuật k-nearest neighbors, thường được biết đến là k-NN, là cách tiếp cận phi-tham số, ở phương pháp này phân lớp của một điểm dữ liệu được định nghĩa bởi k điểm dữ liệu gần nó nhất trong tập huấn luyện. Phương pháp này có thể được sử dụng trong quá trình thiết lập cho bài toán phân loại cũng như bài toán hồi quy.
+
+<br>
+
+**74. Remark: The higher the parameter k, the higher the bias, and the lower the parameter k, the higher the variance.**
+
+&#10230; Chú ý: Tham số k cao hơn, bias cao hơn, tham số k thấp hơn, phương sai cao hơn
```

Comment:
You can translate 'bias' to 'độ chệch'. I found [AIVIVN](https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation) was using it.

Reply:
review accepted